iaac/terraform/kubeflow-components/dex/locals.tf

```diff
@@ -0,0 +1,17 @@
+locals {
+  name = "dex"
+
+  default_helm_config = {
+    name        = local.name
+    version     = "0.1.0"
+    namespace   = "default"
```

Comment:
what is this namespace? lets check later on this, I think the helm install is also happening in default namespace, we should change this to whatever namespace the resources are created

Reply:
adding todo

Reply:
done